functions/public/Invoke-WPFGetIso.ps1

```diff
@@ -84,11 +84,26 @@ function Invoke-WPFGetIso {
 
         Invoke-WebRequest "https://github.com/pbatard/Fido/raw/master/Fido.ps1" -OutFile $fidopath
 
-        # TODO: Have to correct the language to actually be a working one
+        # TODO: on get iso, script gets downloaded and function is taken & called to get & display languages. after selection user can press proceed, which will start the fido process and select the output iso for further processing
         Set-Location -Path $env:temp
-        & $fidopath -Win 'Windows 11' -Rel $sync["ISORelease"].SelectedItem -Arch "x64" -Lang "$(Get-FidoLangFromCulture -langName `"$($sync.ISOLanguage.SelectedItem)`")"
+        & $fidopath -Win 'Windows 11' -Rel $sync["ISORelease"].SelectedItem -Arch "x64" -Lang $sync["ISOLanguage"].SelectedItem
```

Comment:
@MyDrift-user, I don't think Fido works like that. I don't think you can pass en-US to it to download the English version

Reply:
@CodingWonders yeah this is a WIP push bc I work on multiple clients without shared storage...

I want to remove the Locale selection (currently) bc it is not needed. I am working on making the dropdown select languages, which then are passed to fido. This is a neccessary step and therefore more relevant.